custom/icds_reports/views.py

```diff
@@ -1815,6 +1816,16 @@ def get(self, request, *args, **kwargs):
         return JsonResponse({'message': 'Connection Successful'})
 
 
+@location_safe
+@method_decorator([api_auth], name='dispatch')
+class DailyIndicators(View):
+    def get(self, request, *args, **kwargs):
```

Comment:
Is this view like the aww_inactive one, where it is just supposed to be directly hit? If so, why does it use API auth?

Reply:
Ah!! yeah Good catch. it has to be `login_and_domain_required`